doc/api.mld

```diff
@@ -0,0 +1,22 @@
+{0 API}
+
+The entrypoint for OCamlFormat's API is the {{:https://v3.ocaml.org/p/ocamlformat/0.21.0/doc/Ocamlformat/Translation_unit/index.html}Translation_unit} module.
```

Comment:
That's not right. Documentation should be self-contained, also this link is meaningless once this documentation is hosted on v3.
```suggestion
OCamlformat's library API: {!Ocamlformat}. The entrypoint is the {!Ocamlformat.Translation_unit} module.
```